blocks/library/pullquote/style.scss

```diff
@@ -21,15 +21,15 @@
 		line-height: 1.6;
 	}
 
-	footer {
+	cite {
 		color: $dark-gray-600;
 		position: relative;
 		font-weight: 900;
 		text-transform: uppercase;
 		font-size: 13px;
 	}
 
-	footer p {
+	cite p {
```

Comment:
`cite` can only contain phrasing content, so this doesn't look right.

Reply:
Ah, you're right. However, I just adapted the CSS rules that were there. Currently, a paragraph within a `<cite>` element is not used anywhere.

Should I just remove that rule then, @iseulde?

Reply:
I don't know, would need to look into it. Does the cite element still contain paragraphs? Not sure why that rule was there.

Reply:
Looks like you can just remove it.

Reply:
Like I said, no it does not. I am talking about all the fixtures here. Also, the regular _quote_ does not have this rule, only the _pullquote_. Also also, seeing `<Editable tagName="cite" value={ citation } ...`, I'm not sure if paragraphs would actually work. I'd say the citation is not designed to contain more than a reference (and thus, of course, not something in multiple paragraphs).

Reply:
Right, if it were it would be `<Editable multiline="true" />`.

Reply:
Removed.